data_acquisition/py/gee_functions.py

```diff
@@ -373,36 +491,154 @@ def ref_pull_457_DSWE1(image):
                                   'mean_SurfaceTemp']))
             .addBands(image.select(['SurfaceTemp']))
             .updateMask(d.eq(1)) # only high confidence water
-            .updateMask(r.eq(1)) #1 == no saturated pixels
-            .updateMask(f.eq(0)) #no snow or clouds
-            .updateMask(s.eq(0)) # no SR processing artefacts
             .updateMask(hs.eq(1)) # only illuminated pixels
```

Comment:
```suggestion
            .updateMask(r.eq(1)) #1 == no saturated pixels
            .updateMask(f.eq(0)) #no snow or clouds 
            .updateMask(hs.eq(1)) # only illuminated pixels
```

Reply:
Add this back in, though logically, this should work, i'm getting dswe counts of 0 and band data..... #facepalm

Reply:
Need to do this across all functions!